ui/pages/swaps/searchable-item-list/index.scss

```
@@ -170,7 +170,7 @@
     @include H7;
 
     line-height: 100%;
-    color: var(--Grey-500);
+    color: var(--color-text-muted);
```

Can probably use text-alternative here
```suggestion
    color: var(--color-text-alternative);
```